package.json

```diff
@@ -12,10 +12,12 @@
     "style": "yarn format && yarn lint"
   },
   "dependencies": {
-    "@auth0/nextjs-auth0": "^1.8.0",
+    "@auth0/nextjs-auth0": "^2.2.1",
```

Comment:
While our code will work with this version of next auth, certain bit.dev components OCF has provided required the version we currently have in the repo. For now, I think we should keep it downgraded in case we need to use these, and consider upgrading later.

Make sure to run yarn install after changing this back.